recipes-core/acrn/acrn-tools/0001-meta-acrn-do-not-build-life_mngr-by-default.patch

```diff
@@ -0,0 +1,49 @@
+From 823f5b3696d88cd0eb24c6b3f47391f0244616f9 Mon Sep 17 00:00:00 2001
+From: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>
+Date: Wed, 3 Mar 2021 10:24:38 +0100
+Subject: [PATCH] meta-acrn: do not build life_mngr by default
+
+Do not build (or install) the ACRN life_mngr by default as this is
+a User VM tool, not one to be used and run in the Service VM.
+
+The component can still be built independantly by invoking
+'make -C misc life_mngr' (components will be built and placed in
+'misc/build/services/' by default).
+
```

Comment:
Could you please add Upstream-status to this patch  ? 

Upstream-Status: Inappropriate [Build Specific or other reason]

Ref: https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines

Reply:
> Could you please add Upstream-status to this patch ?

Done. I used `Pending` though as I believe this should be submitted upstream. And it is my intention to do it ;-)

Reply:
> Done. I used `Pending` though as I believe this should be submitted upstream. And it is my intention to do it ;-)

FYI - https://github.com/projectacrn/acrn-hypervisor/pull/5814